src/web/src/sse.cc

```diff
@@ -299,7 +299,8 @@ std::string Sse::changeFtyProtoAsset2Json(fty_proto_t *asset)
       {
         //Check if the asset is a device 
         log_debug("Asset type : %s",fty_proto_aux_string(asset,"type","none"));
-        if (streq(fty_proto_aux_string(asset,"type","none"),"device")){
+        if (streq(fty_proto_aux_string(asset,"type","none"),"device") ||
+            streq(fty_proto_aux_string(asset,"type","none"),"none")){
           //Add in the list of asset without location, will send a normal create message
```

Comment:
Now there are three calls that retrieve the asset type field. Is it possible to do one call and store it in a variable? Also, the commend still says "Check if the asset is a device" but we are also taking assets with no type field now.